packages/editor/src/toolbar/popups/color-picker.tsx

```diff
@@ -220,6 +268,17 @@ export function ColorPicker(props: ColorPickerProps) {
                 }
               }}
               onClick={() => {
+                if (trashMode) {
+                  if (colors.indexOf(color) > 13) {
```

Comment:
What does `> 13` mean? If this is used to detect whether the color is a default one or not...how about just checking the `DEFAULT_COLORS` instead?